src/vectored_io.rs

```diff
@@ -32,10 +32,9 @@ impl<'io, const N: usize> VectoredWriter<'io, N> {
         for buf in self.data[self.skip..].iter() {
             if accumulated_len + buf.len() > len {
                 break;
-            } else {
-                accumulated_len += buf.len();
-                remove += 1;
             }
+            accumulated_len += buf.len();
```

Comment:
I personally prefer the unnecessary `else` here, as that makes human errors less likely as removing the break for whatever reason doesn't change the behaviour.